_docs-sources/guides/stay-up-to-date/index.md

```diff
@@ -8,7 +8,19 @@ import CardGroup from "/src/components/CardGroup"
 
 # Stay Up To Date
 
-<Grid cols={2} gap="4rem">
+<Grid cols={3} gap="4rem">
+
+<span>
+
+## Gruntwork releases
+
+<CardGroup cols={1} gap="1rem" stacked equalHeightRows={false} commonCardProps={{padding: "1.25rem"}}>
+
+<!-- replaced-by-docs-sourcer-automatically-do-not-edit -->
```

Comment:
I think this is OK for now - but I really want to rethink where these release notes will live going forward.

I have 2 unrelated concerns here:
- We didn't _really_ consider the case where there would be "mixed" content in the manually edited code. At least not in the way that you're using it here. We did plan for the case where we _pull in_ content from remote sources (ie - code blocks that reference remote content). It's possible that your solution here is just fine. Ie - the plugins should be through of as a "pipeline" which has steps executed in order. We should give this a bit more thought before buying into this
- My other concern is that this just isn't the right place for the page. I think that concern can be addressed with future updates. Worst case scenario if we end up moving this content around I think we can just update the published blog post to correct the URLs (rather than dealing with possible lots and lots of redirects).

Reply:
> * We didn't _really_ consider the case where there would be "mixed" content in the manually edited code. At least not in the way that you're using it here. We did plan for the case where we _pull in_ content from remote sources (ie - code blocks that reference remote content). It's possible that your solution here is just fine. Ie - the plugins should be through of as a "pipeline" which has steps executed in order. We should give this a bit more thought before buying into this

Roger.

> * My other concern is that this just isn't the right place for the page. I think that concern can be addressed with future updates. Worst case scenario if we end up moving this content around I think we can just update the published blog post to correct the URLs (rather than dealing with possible lots and lots of redirects).

Fine by me! Note that we should probably start linking to these releases pages from other places in our docs too: e.g., our intro and Ref Arch docs should probably have some mention of how to keep your code up to date, and a link to these releases pages to see those releases. So if we're going to move these pages to new URLs, we either need to do it ASAP, so there aren't many inbound links to change, or we will have to go through the redirect work.